website/templates/emails/mentions.txt.mako

```diff
@@ -0,0 +1,3 @@
+${user} at ${localized_timestamp}: ${content}.
```

Comment:
They get stored and sent out by `send_users_email`. The email flow wasn't altered.